### PR TITLE
136 edit platform modal pending

### PR DESCRIPTION
**Pull request by philboeselager**

### CODE REVIEW

**Comment by philboeselager** [comment]
depends on https://github.com/openlibraryenvironment/gokb/pull/496

File: src/shared/services/platform-services/platform-services.js [hunk 1]
```diff
@@ -15,6 +15,20 @@ const api = (baseServices) => ({
...
       data,
     }, cancelToken)
+  },
+  checkPlatformFields (data, fieldList, cancelToken) {
+    const filteredData = Object.keys(data)
+    .filter(key => fieldList.includes(key))
+    .reduce((obj, key) => {
+      obj[key] = data[key];
+      return obj;
+    }, {});
```
Comment (hornmo) [inline src/shared/services/platform-services/platform-services.js]: I don't think we need this filtering via fieldList here. Just put only name & primaryUrl into `data` and create the params from that.

**Review by hornmo — CHANGES_REQUESTED** [review]
There seems to be an issue with the displaying of the error messages:

`<v-col v-if="c.id">
 {{ $t('component.platform.conflict.noProvider', [c.id]) }}` (line 48/49)

Should be for `!c.id` (no provider id) and the parameter should be `c.platformId` instead of `c.id`.

File: src/shared/services/platform-services/platform-services.js (hunk 1 above)
Comment (philboeselager) [reply src/shared/services/platform-services/platform-services.js]: True. Came out of the idea if we'd need a such functionality in other methods, but it's pure overload. Has been simplified now.

**Comment by philboeselager** [comment]
> There seems to be an issue with the displaying of the error messages:
> 
> `<v-col v-if="c.id"> {{ $t('component.platform.conflict.noProvider', [c.id]) }}` (line 48/49)
> 
> Should be for `!c.id` (no provider id) and the parameter should be `c.platformId` instead of `c.id`.

Changed.

**Review by hornmo — APPROVED** [review]

**Pull request merged**